typescript/sdk/src/consts/chainMetadata.ts

```diff
@@ -689,6 +689,23 @@ export const zbctestnet: ChainMetadata = {
   isTestnet: true,
 };
 
+export const zkevm: ChainMetadata = {
```

Comment:
I agree with Nam's point from his review, I'm not certain that including a PI chain here as a 'core' chain is the right thing. For example, it's inclusion here will fool the Explorer/Warp UI into thinking it's not a PI message, and expect it to have scraped db data. Will that be the case? 

If you're aware of the downside and still prefer to include it, I'm okay with it